src/styles/_colors.scss

```diff
@@ -0,0 +1,20 @@
+// Primary palette
+$primaryGreen: #008254;
+$primaryBlack: #1b1b1b;
+$primaryOffWhite: #fafafa;
+$primaryWhite: #ffffff;
+
+// Secondary palette
+$secondaryGreen: #e3f9f1;
+$secondaryYellow: #feefd0;
+$secondaryBlue: #deeffe;
+$secondaryGray: #f2f2f2;
+$darkGray: #555555;
+$secondaryWhite: #e5e5e5;
```

Comment:
This is the page background color in most of the mocks in Figma, but it doesn't appear in the palettes under Design System. I'm not sure if this is actually supposed to be $primaryWhite, or if we need to keep it separate.

Reply:
Need to update or ready for merge?

Reply:
@JMStudiosJoe Ok to merge, I was just curious if anyone knew whether that color should be used as primary white, or if it's only for backgrounds. But I can always change it later if needed.

Reply:
The Design System section on Figma defines primary white as #FFFFFF, so I think we're good on that front!